src/bandersnatch_filter_plugins/blacklist_name.py

```diff
@@ -72,26 +71,55 @@ def check_match(self, **kwargs):
         bool:
             True if it matches, False otherwise.
         """
-        logger.debug('Running filter plugin %s', self.name)
         name = kwargs.get('name', None)
         if not name:
             return False
 
-        logger.info(
-            'Checking for package %s in %r', name, self.blacklist_package_names
-        )
         if name in self.blacklist_package_names:
-            logger.debug(
-                'MATCH: Package %r is in %r', name,
-                self.blacklist_package_names
-            )
+            logger.debug(f'MATCH: Package {name!r} is in the blacklist')
             return True
         return False
 
 
 class BlacklistRelease(FilterReleasePlugin):
     name = 'blacklist_release'
 
+    def initialize_plugin(self):
+        """
+        Initialize the plugin
+        """
+        # Generate a list of blacklisted packages from the configuration and
+        # store it into self.blacklist_package_names attribute so this
+        # operation doesn't end up in the fastpath.
+        self.blacklist_release_requirements = \
```

Comment:
I prefer using () over \ for line continuation - so does the `black` code style which I will soon be converting this code base too and adding a black run into travis

Reply:
Unfortunetly, the ( doesn't occur in the line then until after 80 columns.